upgrade/sql/9.0.0.sql

```diff
@@ -323,3 +323,106 @@ ALTER TABLE `PREFIX_attachment_lang` MODIFY COLUMN `name` varchar(255) DEFAULT N
 /* Fix category thumbnail images */
 /* https://github.com/PrestaShop/PrestaShop/pull/36877 */
 /* PHP:ps_900_migrate_category_images(); */;
+
+/* Auto generated hooks added for version 9.0.0 */
+INSERT INTO `PREFIX_hook` (`id_hook`, `name`, `title`, `description`, `position`) VALUES
```

Comment:
Be careful the list of inserted hooks is already upper in this file The command is able to generate the SQL query but it cannot modify this file "in place", it only appends the content at the end So we need to manually keep the file consistent (either move this added code upper in the file or remove the upper part)